move/walrus_site/sources/site.move

```diff
@@ -26,7 +28,12 @@ module walrus_site::site {
         blob_id: u256,
         // Contains the hash of the contents of the blob
         // to verify its integrity.
-        blob_hash: u256
+        blob_hash: u256,
+        // Defines the byte range of the resource contents
+        // in the case where multiple resources are stored
+        // in the same blob. This way, each resource will
+        // be parsed using its' byte range in the blob.
+        range: Option<vector<u256>>,
```

Comment:
Question: why use an `Option<vector<u256>>`? Can we use tuples? I don't remember if move allows them. 

Suggestion: If it does not, we could strongly type it instead as `Range { start: Option<u256>, end: Option<u256> }`. Then, add an `Option<Range>` to the `Resource`. This may be a bit more readable and easy to parse? And also we don't need the length checks.

Reply:
Yes as far as I know there is no tuple definition in move. Sure I can use a Range struct 🙆‍♂️ !